styles/widgets/material/color-schemes/material.light.less

```diff
@@ -1458,7 +1458,7 @@
 * @type color
 * @group overlays.tooltip
 */
-@tooltip-bg: #616161;
+@tooltip-bg: rgba(255, 255, 255, 1);
```

Comment:
Look at my comment about charts' tooltip. Material tooltip background should be #616161.